README.md

```diff
@@ -1,76 +1,130 @@
 ![](workshops/static/amy-logo.png)
 
-[Software Carpentry](http://software-carpentry.org) is now running three events a week.
-That number could double by the end of 2015
-but only if we can
+[Software Carpentry](http://software-carpentry.org) is running three events
+a week.  The number could double by the end of 2015, but only if we can
 [streamline setup and organization](http://software-carpentry.org/blog/2014/12/plans-for-2015-workshop-organization.html).
-The goal of this project is to build
-a web-based workshop administration application using Django to do that.
-It is not intended to be visible to the general public,
-or even to instructors (at least initially --- we may add that later).
-Instead,
-the target audience is administrators,
-most of whom are non-programmers,
-who need to keep track of
-what workshops are being arranged,
-when they're supposed to occur,
-who's teaching what,
-and so on.
-
-To get started:
-
-1.  Install Django and its dependencies.
+The goal of this project is to build a web-based workshop administration
+application using Django to do that.  It is not intended to be visible to the
+general public, or even to instructors (at least initially — we may add that
+later).
+
+Instead, the target audience is administrators, most of whon are
+non-programmers, who need to keep track of what workshops are being arranged,
+when they're supposed to occur, who's teaching what, and so on.
+
+# Getting started
+
+1. Clone the repository:
+
+    ~~~
+    $ git clone https://github.com/swcarpentry/amy.git
+    $ cd amy
+    ~~~
+
+2. Install Django and its dependencies:
 
     ~~~
     $ sudo python3 -m pip install -r requirements.txt
     ~~~
 
-2. Install [Bower](http://bower.io/) - the tool that manages Amy's JavaScript and CSS dependencies:
+    If you're experienced Python programmer, feel free to create a Python3-compatible virtualenv for Amy and install dependencies from `requirements.txt`.
+
+3. Install [Bower](http://bower.io/) — the tool that manages Amy's JavaScript and CSS dependencies:
 
     ~~~
-    $ npm install -g bower
+    $ sudo npm install -g bower
```

Comment:
On Tue, May 12, 2015 at 01:01:14PM -0700, Piotr Banaszkiewicz wrote:
> >  
> >      ~~~
> > -    $ npm install -g bower
> > +    $ sudo npm install -g bower
> 

> While I agree with you that we should not use root account for
> security, I also know that to access `bin` under either local npm
> installation dir, or local pip installation dir, it would require
> changing $PATH, adding something to either .bashrc/.zshrc, etc, etc…

Right [1].  I guess that maybe makes it too complicated for our docs.

> I'd leave `sudo` where it is and provide a link to some resources
> for people who choose not to use it.

Fair enough.  Or just use sudo, and trust folks who don't like that to
lookup alternatives on their own.

[1]: https://github.com/wking/dotfiles-public/blob/980cb1cbaf67b9f060acb588d8984ce35347b42f/src/.bash_profile#L13